tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38
+envlist = py37,py38,py39,py310
```

Comment:
I think we should stick to a specific version of python (the one used at runtime) as this is not a library. I know we use it as a library and I think that is bad and we should stop doing that. Also, having a single python version here does not stop one from running it on multiple python versions, but by default it runs on the proper python version

Reply:
This was due to my recommendation. Please see my related comments on this in the gitlab yaml.

Reply:
reset